### ISSUE TITLE
The realism loss does not converge

### ISSUE THREAD

**Issue by MinglangQiao** [compphoto/RealisticImageEnhancement#1]
Hi, thanks for sharing your great work! I followe the instructions and default settings of this repository to train the EditingNet, but I found that the realism loss does not converge and the output images are not  realistic, though the saliency loss converges normally. Could you please give me some suggestions on training the EditingNet? Thanks a lot !

The following figures are training loss and output images:
 
![loss](https://github.com/compphoto/RealisticImageEnhancement/assets/13742613/88325c50-a915-4baf-84b5-77519df61465)
![rel_out1](https://github.com/compphoto/RealisticImageEnhancement/assets/13742613/2c61d7fe-5662-4a66-b935-811b0322367c)
![rel_out2](https://github.com/compphoto/RealisticImageEnhancement/assets/13742613/324cb7f7-b717-40cf-8d46-ba6aa1b1063d)

**Comment by miangoleh** [comment]
I would suggest experimenting with a higher weight for realism loss (Although the weights I shared in the repo should replicate the same results as the model weight I shared and used for the paper)

**Comment by MinglangQiao** [comment]
Thank you for your suggestion; I'll give it a try.
